src/main/java/org/aleksey/springnew/repoitory/TodoHistoryRepository.java

```diff
@@ -0,0 +1,13 @@
+package org.aleksey.springnew.repoitory;
+
+import org.aleksey.springnew.model.Todo;
+import org.aleksey.springnew.model.TodoHistory;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+
+import java.util.List;
+
+public interface TodoHistoryRepository extends JpaRepository<TodoHistory, Long> {
+    @Query("select td from TodoHistory td where td.todoId = ?1")
```

Comment:
```suggestion
    @Query("select th from TodoHistory th where th.todoId = ?1")
```
You are working with `TodoHistory` here, so it is better to use `th` not `td`.